frank-doc-doclet/src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -233,31 +235,56 @@ public static String jsonPretty(String json) {
 	}
 
 	public static String flattenJavaDocLinksToLastWords(String text) throws FrankDocException {
-		return replacePattern(text, JAVADOC_LINK_START_DELIMITER, JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, Utils::getLinkReplacement);
+		return replacePattern(text, JAVADOC_LINK_START_DELIMITER, Utils::getLinkReplacement);
 	}
 
-	private static String replacePattern(String text, String patternStart, String patternStop, Function<String, String> substitution) throws FrankDocException {
+	public static String replacePattern(String text, String patternStart, Function<String, String> substitution) throws FrankDocException {
 		if (text == null) {
 			return null;
 		}
+
 		StringBuilder result = new StringBuilder();
+
 		int currentIndex = 0;
-		int nextStartIdx = text.indexOf(patternStart, currentIndex);
-		while (nextStartIdx >= 0) {
-			result.append(text.substring(currentIndex, nextStartIdx));
-			int endIdx = text.indexOf(patternStop, nextStartIdx);
-			if (endIdx < 0) {
-				throw new FrankDocException(String.format("Unfinished JavaDoc {@ ...} pattern text [%s] at index [%d]", text, nextStartIdx), null);
+		int startIndex = text.indexOf(patternStart);
+		while (startIndex >= 0) {
+			int stopIndex = text.indexOf("}", startIndex);
```

Comment:
```suggestion
			int stopIndex = text.indexOf(JAVADOC_SUBSTITUTION_PATTERN_STOP_DELIMITER, startIndex);
```
Wel zo netjes toch de constanten te gebruiken ;)